util.go

```diff
@@ -1,20 +0,0 @@
-package iface
-
```

Comment:
What's up with the changes to this file, is this just an unrelated cleanup?

Reply:
Ah, yes. I couldn't find any users of this.